### PR TITLE
Added bugfix for Issue #734

### PR DESCRIPTION
**Pull request by rwolf449**
If you close the app on Windows right now, there are several processes still running.
Buttercup does not close properly which leads to the behaviour described in the issue.
I would suggest this small change, Buttercup now closes as intended (tested on Windows and Mac).

### CODE REVIEW

File: src/main/app.js [hunk 1]
```diff
@@ -147,10 +145,7 @@ app.on('ready', async () => {
   // When user closes all windows
   // On Windows, the command practice is to quit the app.
   app.on('window-all-closed', () => {
-    if (
-      appTriedToQuit ||
```
Comment (sallar) [inline src/main/app.js]: Why is this deleted? this is important for saving unsaved data. Check the rest of the file for this `appTriedToQuit`

Comment (rwolf449) [reply src/main/app.js]: I added appTriedToQuit again, and it still works, the checks also passed now. The problem seems to be `!getSetting(store.getState(), 'isTrayIconEnabled'`

Comment (sallar) [reply src/main/app.js]: @rwolf449 Then we have to figure out a way to fix it, not remove it. Since all OS's rely on this setting to NOT quit if the tray icon is enabled.

**Comment by sallar** [comment]
Hey @rwolf449 any updates on this by any chance?

**Comment by perry-mitchell** [comment]
@sallar Should we close this?

**Comment by sallar** [comment]
Yes @perry-mitchell unfortunately this is not a solution to the problem. Apologies @rwolf449
